bql/grammar/grammar_test.go

```diff
@@ -453,6 +459,18 @@ func TestRejectByParse(t *testing.T) {
 			?s ?p ?o .
 			FILTER late^st(?p)
 		 };`,
+		`select ?a
+		 from ?b
+		 where {
+			?s ?p ?o .
+			FILTER greaterThan(?o "37"^^type:int64)
+		 };`,
```

Comment:
I guess I'm missing something.
How come this test is on both TestRejectByParse and TestAcceptByParse?

Reply:
In `TestRejectByParse` there is a comma missing between the binding and the value.

Reply:
heauehaueh, I would never notice this.
Thanks!